### PR TITLE
Remove martha_v1 [WA-14]

### PR DESCRIPTION
**Pull request by salonishah11**
JIRA issue [here](https://broadworkbench.atlassian.net/browse/WA-14).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,13 +6,7 @@ Martha
...
 
 # Martha v1
-To call `martha_v1`, perform an HTTP `POST` to the appropriate URL. The body of the request must be a JSON Object with
-two values: a [DOS](https://data-object-service.readthedocs.io/en/latest/) URL and a regex pattern. Martha uses the
-DOS URL to retrieve a data object, unpacks it, and returns the first link that matches the specified regex pattern.
-For more details look [here](https://docs.google.com/document/d/1AyyI6L43te_DFWh8dXAiX0Qx-8f3JLKUIZe6xFwKMb0/edit#).
-
-Staging: https://us-central1-broad-dsde-staging.cloudfunctions.net/martha_v1
-Production: https://us-central1-broad-dsde-prod.cloudfunctions.net/martha_v1
+Deprecated as of March 2020. Please use Martha v2.
```
Comment (kshakir) [inline README.md]: Deprecation usually implies that functionality still works. In this case, is the function being left behind, or removed from GCF?

Comment (salonishah11) [reply README.md]: No. We are deleting the function. Should this message be updated to say `Deleted as of ...`?

Comment (kshakir) [reply README.md]: Yep, sounds good to me

**Review by kshakir — APPROVED** [review]

**Comment by salonishah11** [comment]
Jenkins automation is currently failing because in fiab-start, it is waiting for martha_v1 to respond which is not going to happen. Once [this](https://github.com/broadinstitute/firecloud-develop/pull/2068) PR is merged, it should be able to start the fiab.

**Comment by salonishah11** [comment]
jenkins retest

**Review by rsasch — APPROVED** [review]

**Pull request merged**